roles/cifmw_ceph_spec/templates/initial_ceph.conf.j2

```diff
@@ -7,7 +7,9 @@ ms_mon_client_mode = secure
 ms_mon_cluster_mode = secure
 ms_mon_service_mode = secure
 {% endif %}
+{% if cifmw_cephadm_single_host_defaults %}
 osd pool default size = 1
+{% endif %}
```

Comment:
This will affect other jobs so let's invert the logic please (per our discussion)

Reply:
Done